GitCommands/ExternalLinks/ExternalLinkRevisionParser.cs

```diff
@@ -99,18 +99,16 @@ private static IEnumerable<ExternalLink> ParseRevision(GitRevision revision, Ext
 
             if (definition.SearchInParts.Contains(ExternalLinkDefinition.RevisionPart.LocalBranches))
             {
-                foreach (var head in revision.Refs.Where(b => !b.IsRemote))
-                {
-                    links.Add(ParseRevisionPart(revision, definition, remoteMatch, head.LocalName));
-                }
+                links.AddRange(revision.Refs
+                    .Where(b => !b.IsRemote)
+                    .Select(head => ParseRevisionPart(revision, definition, remoteMatch, head.LocalName)));
```

Comment:
I don't like this indentation - it gives impression `.Where` and `.Select` apply `links`.
Same applies to the change below.

Reply:
Thanks. I've pushed an improvement.